tests/gwas-tar-gz/hdl_ldl_cholesterol_tar_gz.yml

```diff
@@ -0,0 +1,58 @@
+data_file:
+    description: >-
+        Global Lipid Genetics Consortium GWAS results for high-density
+        cholesterol levels
+    location:
+        url: https://zenodo.org/record/3345991/files/gwas_hdl_ldl.tar.gz
+        md5: eccbd3b5b6ff87e78063321846b78dfa
+    columns:
+        variant_id: panel_variant_id
+        variant_chrom: chromosome
+        variant_pos: position
+        rsnum: variant_id
+        ref: na
+        alt: na
+        effect_allele: effect_allele
+        non_effect_allele: non_effect_allele
+        inc_allele: na
+        inc_afrq: na
+        effect_size: effect_size
+        standard_error: standard_error
+        zscore: zscore
+        pvalue: pvalue
+        feature_id: na
+        feature_chrom: na
+        feature_start: na
+        feature_stop: na
+        imputation_status: imputation_status
+        frequency: na
+        tss_distance: na
+        ma_samples: na
+        maf: na
+        comment_0: na
+
+data_info:
+    citation: 10.1038/ng.2797
+    data_source: http://lipidgenetics.org/
+    data_type: gwas
+    build: b38
+    context: tarball hdl/ldl test
```

Comment:
i am currently updating doc for this but context for a tarball should be a list of traits delimited by ";" so for this example, it would be "hdl cholesterol; ldl cholesterol"